orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -8747,6 +8747,57 @@ definitions:
     - macs
     - ips
     type: object
+  log:
```

Comment:
Any specific reason for renaming this from `message` to `log`?

Reply:
yes, the message in question is called "log" all throughout the code and only here was it called "message", causing some confusion and making it difficult to find the swagger object behind it

Reply:
Got it.